Makefile

```diff
@@ -145,6 +145,11 @@ project-create-ecr:
 dev-setup:
 	make python-virtualenv
 
+dev-clean:
+	make python-virtualenv-clean
+	cd $(APPLICATION_DIR)
+	rm -rfv static/
```

Comment:
I wonder if that should be rather a part of the project clean target? If we think we should keep it here as well, can we convert it to `rm -rf $(APPLICATION_DIR)/static`? One line and no `v` flag as the list isn't that important. What do you think Matt?

Reply:
good idea, I'll change it 👍